### PR TITLE
[#27101] Init Project

### PR DESCRIPTION
**Pull request by kguen-sun**
[Redmine](https://edu-redmine.sun-asterisk.vn/issues/27101)

### CODE REVIEW

File: config/credentials.yml.enc [hunk 1]
```diff
@@ -0,0 +1 @@
+JuaEReyM5Vjcfc3AuG5KTuLHDCIyXboadBIlPz6BWmw2s/jdlzxgkrFfU7BzODq/WQx0VV8V1tCLkWHBJ3gf5K4jYJgD8JCgUHoIrpOdspiv9OPYq8vvBW/dDVXAxMWGKB9PQCAaL83jQql/tRr6ryeNYozfLywdefGxE1FpY6R6mjTq6KJQJRbxUgmWAHJbAJWeY52Z4QVylZZBndAmmTGyVshY/jBIbuX510oiT7X7rJDaKszKtPi4ZBAGGkBU+dhI55rJbPdcp6q2YZund8FeGS/aA4plooDq1gq/Db32SaVi5RCcH5whQKHFtoNyLGdzVCtNVvcL+obN/TnWDRwBxuyxSQX42ginFgjBYKDx69voJET0+RDOzmyZoDYxx6kx0F+yj74m5iwZ/5b9meNzoNOPnBgVh/HO--LP8T0dqDaFk+uXgx--0PRMDbvRlwFg0xaSwGYOKw==
```
Comment (nthtra-110273) [inline config/credentials.yml.enc]: thêm dòng trống

File: config/database.yml.example [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: store_management_test
+
+production:
+  <<: *default
+  database: store_management_production
+  username: store_management
+  password: <%= ENV['STORE_MANAGEMENT_DATABASE_PASSWORD'] %>
```
Comment (nthtra-110273) [inline config/database.yml.example]: Thêm dòng trống =))

**Comment by kguen-sun** [comment]
Screenshot from 2020-08-13 15-17-00
Screenshot from 2020-08-13 15-43-31
Screenshot from 2020-08-13 15-47-20
Screenshot from 2020-08-13 15-47-57

**Review by nguyenvd16 — APPROVED** [review]

**Review by nthtra-110273 — APPROVED** [review]

**Comment by kguen-sun** [comment]
ready

**Pull request merged**